src/map.rs

```diff
@@ -230,7 +230,11 @@ impl Clone for Map<String, Value> {
 impl PartialEq for Map<String, Value> {
     #[inline]
     fn eq(&self, other: &Self) -> bool {
-        self.map.eq(&other.map)
```

Comment:
Please keep the existing implementation if we are building without preserve_order. I believe BTreeMap will be able to determine equality significantly faster.

Reply:
Fixed